Validation/Performance/python/TimeMemorySummary.py

```diff
@@ -15,8 +15,8 @@ def customise(process):
     if hasattr(process,"options"):
         process.options.wantSummary = cms.untracked.bool(True)
     else:
-        process.options = cms.untracked.PSet(
-            wantSummary = cms.untracked.bool(True)
+        process.options = dict(
+            wantSummary = True
             )
```

Comment:
This could become nowadays just
```py
    process.options.wantSummary = True
```

Reply:
@makortel Thank you for pointing this. Let me update this!